Clarify that deployments should live in each repo in a subdir of `deployments` (#345)
signalfx/gdi-specification: specification/repository.md

  - SHOULD define maximum supported version of each auto-instrumentation framework
- MAY host the documentation in the [Observability Cloud documentation repository](https://github.com/splunk/public-o11y-docs)

## Deployments

For this section, "deployments" refers to software tools, utilities, and/or
configuration for packaging, distributing, and/or installing `splunk-otel`
GDI software. Typically, these are 3rd-party frameworks like ansible, puppet,
chef, cloudfoundry, etc.
The `splunk-otel-collector`

> [distributions](https://github.com/signalfx/splunk-otel-collector/tree/main/deployments)

Reviewer: ```suggestion
[deployments](https://github.com/signalfx/splunk-otel-collector/tree/main/deployments)
```
no?

Author: yes, good catch, thanks!

are a good example of this.

> - Splunk O11y GDI repos MAY include one or more distributions.

Reviewer 2: I get what you mean by `distributions` here but I'm not sure it is a commonly used work in this context.

Author: I meant deployments! Doh. Fixing....

- The distributions for a given GDI project MUST each exist within a separate

>   subdirectory of a repo root directory named `distributions`.

Reviewer: ```suggestion
  subdirectory of a repo root directory named `deployments`.
```

> For this section, "deployments" refers to software tools, utilities, and/or
> configuration for packaging, distributing, and/or installing `splunk-otel`
> GDI software. Typically, these are 3rd-party frameworks like ansible, puppet,
> chef, cloudfoundry, etc.
> The `splunk-otel-collector`
> [deployments](https://github.com/signalfx/splunk-otel-collector/tree/main/deployments)
> are a good example of this.
> 
> - Splunk O11y GDI repos MAY include one or more deployments.
> - The deployments for a given GDI project MUST each exist within a separate
>   subdirectory of a repo root directory named `deployments`.
> - An historical exemption is provided for `splunk-otel-collector-chart`, which
>   MAY live in a separate repository.

Reviewer 3: nit

```suggestion
For this section, "deployments" refers to software tools, utilities, and
configuration for packaging, distributing, and installing GDI software.
Typically, these are 3rd-party frameworks like Ansible, Puppet, Chef,
CloudFoundry, etc.
The `splunk-otel-collector`
[deployments](https://github.com/signalfx/splunk-otel-collector/tree/main/deployments)
are a good example of this.

- Repository MAY include one or more deployments.
- Each deployment MUST be in its own subdirectory under
  the `deployments` folder at the repository root.
- An historical exemption is provided for `splunk-otel-collector-chart`, which
  MAY live in a separate repository.
```
